docs/UserGuide.adoc

```diff
@@ -241,6 +257,8 @@ e.g. `add n/James Ho p/22224444 e/jamesho@example.com a/123, Clementi Rd, 123466
 e.g. `delete 3`
 * *Edit* : `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]...` +
 e.g. `edit 2 n/James Lee e/jameslee@example.com`
+* *Remove* : `remove TAG` +
+e.g. `remove friend`
 * *Find* : `find KEYWORD [MORE_KEYWORDS]` +
```

Comment:
Documentation updates are fine, but just a general comment - you might want to split the changes into smaller commits next time (just in case you want to `revert` etc.) and to keep track of the incremental changes that happened in this PR